hibernate6-jpa/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'io.micronaut.build.internal.sql-module'
+}
+
+// TODO: Temporary disabled tests and binary compatibility check
+test.onlyIf { false }
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
+    }
+}
+
+dependencies {
+    annotationProcessor(mn.micronaut.graal)
+
+    // included for Java 11
+    runtimeOnly(libs.jaxb.api)
+
+    api(libs.managed.hibernate6.core) {
+        exclude group:'org.jboss.spec.javax.transaction', module:'jboss-transaction-api_1.2_spec'
+        exclude group:'org.javassist', module:'javassist'
+        exclude group:'org.jboss:jandex', module:'jar'
+    }
+    api(libs.jakarta.transaction.api)
+    api(mn.micronaut.validation)
+    // TODO: This would pull in hibernate5 so we want to exclude it for now
+    // Later will include mnData.micronaut.data.tx.hibernate6 after micronaut-data supports Hibernate6
+    api(mnData.micronaut.data.tx.hibernate) {
+        exclude group: 'org.hibernate'
+    }
+    api(mn.micronaut.aop)
+    api projects.jdbc
+    api(mn.micronaut.runtime)
+    api(mn.micronaut.validation)
+
+    compileOnly(libs.graal.svm)
+    compileOnly(libs.managed.hibernate6.micrometer)
+    compileOnly(libs.managed.hibernate6.jcache)
+    compileOnly(mn.micronaut.management)
+    compileOnly(mnMicrometer.micronaut.micrometer.core)
+
+    testAnnotationProcessor(mn.micronaut.inject.java)
+    testRuntimeOnly(libs.micronaut.hibernate.validator)
+    testImplementation(mn.micronaut.inject.java)
+    testImplementation(libs.managed.hibernate6.jcache)
+    testImplementation(mn.micronaut.management)
+    testImplementation(mn.micronaut.inject.groovy)
+    testImplementation(mn.micronaut.function.web)
+    testImplementation(mnCache.micronaut.cache.core)
+    testImplementation(mnMicrometer.micronaut.micrometer.core)
+    testImplementation(libs.managed.hibernate6.micrometer)
+
+    testRuntimeOnly(libs.micronaut.hibernate.validator)
+    testRuntimeOnly projects.jdbcTomcat
+    testRuntimeOnly(libs.managed.h2)
+
+    // TODO: Temporary just to be able to compile tests
+    testCompileOnly(mnData.micronaut.data.tx.hibernate)
```

Comment:
I would remove this dependency for now

Reply:
Yes, I removed all tests for now. Later will re-add from hibernate-jpa and adjust tests.